python/gui/auto_generated/qgstextformatwidget.sip.in

```diff
@@ -145,11 +145,9 @@ Sets the background color for the text preview widget.
 :param color: background color
 %End
 
-    void enableDataDefinedAlignment( bool enable );
+    void updateDataDefinedAlignment();
```

Comment:
@nyalldawson what is the current approach with API changes like this (not often used)?
Can they be documented in api_break.dox or do we need to keep a `( bool enable = false)` and document it as unused param to be removed with QGIS 4?

Reply:
I'd suggest deprecating the old enableDataDefinedAlignment method and making it a no-op. 

(And then hiding the new updateDataDefinedAlignment() method from sip, as neither should be exposed)